src/frontend/Cargo.toml

```diff
@@ -35,6 +35,7 @@ risingwave_source = { path = "../source" }
 risingwave_sqlparser = { path = "../sqlparser" }
 serde_json = "1"
 smallvec = { version = "1.6.1", features = ["serde"] }
+tempfile = "3"
```

Comment:
Because in this part delete the cfg test, so need to import tempfile in depdency.
<img width="649" alt="image" src="https://user-images.githubusercontent.com/36026213/162795977-ed95178d-dafb-4b45-9078-22ae20bf19c3.png">